pkg/constDb/otherConst.go

```diff
@@ -0,0 +1,21 @@
+package constDb
```

Comment:
otherConst not the best name for file. Could you please change it to something better